docs/source/markdown/podman-machine-init.1.md.in

```diff
@@ -149,6 +149,11 @@ options are:
 * **rw**: mount volume read/write (default)
 * **security_model=[model]**: specify 9p security model (see below)
 
+Note: The following destinations are forbidden for volumes: `/bin`, `/boot`, `/dev`, `/etc`,
+`/home`, `/proc`, `/root`, `/run`, `/sbin`, `/sys`, `/tmp`, `/usr`, and `/var`. Subdirectories
+of these destinations are allowed.
```

Comment:
While we do allow subdirectories, it is still not recommend, maybe it should be made clear that mounting over existing important system directories is not supported and can lead to many issues.

Reply:
with the drift of this PR, it has lost some of its meaning.  In the case of /tmp, subdirs are fine as you point out but in that case I have no problem recommending it.  /var on the other hand, yeah ... not a good idea imho